src/internal/_symbolIterator.js

```diff
@@ -0,0 +1 @@
+module.exports = (typeof Symbol !== 'undefined') ? Symbol.iterator : '@@iterator';
```

Comment:
I believe so, see [here](http://kangax.github.io/compat-table/es6/#Symbol).  Although it probably wouldn't hurt to add another check:

```javascript
module.exports = typeof Symbol !== 'undefined' && typeof Symbol.iterator !== 'undefined' ? Symbol.iterator : '@@iterator';
```